bionic/executor.py

```diff
@@ -50,18 +67,38 @@ class MyManager(SyncManager):
         self._logging_receiver = LoggingReceiver(self._logging_queue)
         self._logging_receiver.start()
 
+        self._process_pool_exec = None
+        self.init_or_resize_process_pool(worker_count)
+
+    def init_or_resize_process_pool(self, worker_count):
+        """
+        Initializes the process pool if not initialized yet and
+        returns True.
+
+        If already initialized, resizes the process pool if the
+        worker_count is different than the worker count used
+        previously. When resizing, it returns whether the process
```

Comment:
"When resizing, it returns whether the process pool was resized." This makes it sound like it always return True! Maybe we can just say "... and returns a boolean indicating whether the pool was resized"?